modal.css

```diff
@@ -0,0 +1,280 @@
+  overflow-y: auto;
+  display: flex;
+  flex-direction: column;
+  background-color: white;
+  border: 5px solid #dfe1e6;
+  border-radius: 1em;
+  position: fixed;
+  padding: 2em;
+  z-index: 99;
+  top: 1em;
+  bottom: 1em;
+  right: 1em;
+  left: 1em;
+  width: 365px;
+  height: 90vh;
+}
+
+.button {
+  color: #6070ff;
+  background-color: #ebebff;
+  padding: 4px 12px;
+  font-size: 12px;
+  border-radius: 8px;
+}
+
+.tech_lang {
+  display: flex;
+  flex-direction: row;
+  gap: 10px;
+  margin-top: 30px;
+}
+
+.project-header {
+  color: #172b4d;
+  line-height: 1.75em;
+  font-weight: 700;
+}
+
+.project-header > h2 {
+  font-size: 32px;
+  line-height: 44px;
+  font-weight: 700;
+  margin-bottom: 22px;
+}
+
+.project-header > p {
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+  font-weight: 600;
+  text-transform: uppercase;
+  margin-bottom: 20px;
+}
+
+.pop-img {
+  min-height: 100px;
+  width: 100%;
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.project-desc {
+  margin-bottom: 24px;
+  font-family: 'poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.prfee {
+  display: flex;
+  justify-content: start;
+}
+
+.project-footer,
+.project-footer > div {
+  display: flex;
+  flex-direction: column;
+}
+
+.flex-cards {
+  display: flex;
+  flex-direction: row;
+  gap: 10px;
+  margin: 20px 0 22px 0;
+}
+
+.bullet {
+  font-size: 20px;
+}
+
+.first {
+  font-family: 'poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.second {
+  font-family: 'poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #6b778c;
+}
+
+.project-footer ul {
+  align-self: flex-start;
+  align-items: center;
+  text-align: center;
+  flex-direction: row;
+  display: flex;
+  list-style-type: none;
+  gap: 8px;
+  font-size: 12px;
+  line-height: 1rem;
+  letter-spacing: 0.03em;
+  margin-bottom: 1.5rem;
+}
+
+.project-footer li {
+  background-color: #ebebff;
+  color: #6070ff;
+  border-radius: 8px;
+  padding: 4px 12px;
+  font-weight: 500;
+}
+
+.project-links {
+  display: flex;
+  padding-bottom: 20px;
+}
+
+.project-close {
+  font-size: 22px;
+  color: #868990;
+  float: right;
+  cursor: pointer;
+  border: none;
+  background-color: #fff;
+  transition: 0.4s;
+  margin-bottom: 15px;
+}
+
+.project-close:hover {
+  font-size: 25px;
+  color: #7a9ef0;
+}
+
+.project-links a {
+  font-size: 15px;
+  padding: 10px;
+  background-color: white;
+  border-radius: 8px;
+  color: #6070ff;
+  font-weight: 500;
+  border: 1px solid #7f8cff;
+  margin-right: 10px;
+  width: 175px;
+  text-decoration: none;
+  text-align: center;
+  line-height: 20px;
+}
+
+.project-links a i {
+  margin-left: 10px;
+}
+
+.project-img {
+  min-height: 100px;
+  width: 100%;
+  height: 50%;
```

Comment:
- [ ] I kindly suggest you remove this property of height: 50% as it will make your image take it's full width.